app/components/title.tsx

```diff
@@ -0,0 +1,78 @@
+import type {ReactNode} from 'react'
+import clsx from 'clsx'
+import * as React from 'react'
+
+interface TitleProps {
+  variant?: 'primary' | 'secondary'
+  children: ReactNode
+}
+
+const titleColors = {
+  primary: 'text-black dark:text-white',
+  secondary: 'text-gray-400 dark:text-blueGray-500',
+}
+
+function H1({variant = 'primary', children}: TitleProps) {
```

Comment:
The tag names are extracted from the designs. I'm not sure they reflect the proper semantics. Almost everything on the home page is a heading. What's wisdom here?

Reply:
If it's just styled like a heading but isn't semantically a heading that's fine. Let's just make sure the semantics are correct. Maybe we could support an `as` prop (I normally hate those kinds of props, but I think it makes sense here). So folks could do: `<H1 as="div" ... />`